src/main/kotlin/net/ccbluex/liquidbounce/render/engine/RenderTasks.kt

```diff
@@ -69,48 +69,49 @@ data class Vec3(val x: Float, val y: Float, val z: Float) {
 
 data class UV2f(val u: Float, val v: Float)
 
-data class Color4b(val r: Int, val g: Int, val b: Int, val a: Int = 255) {
+/**
+ * Directly use primitive int value for Color operations.
+ */
+@Suppress("NOTHING_TO_INLINE", "detekt:TooManyFunctions")
+@JvmInline
+value class Color4b private constructor(val argb: Int) {
```

Comment:
What you try to do is a good optimization, it does improve performance and decreases memory usage and fragmentation. But this is a job of the JIT compiler. This class is not even performance critical (which would be an indication for doing such optimizations). 

Inline functions are only useful for inlining parameter functions (i.e. `.forEach { if (it.x > 0) return }`) since it allows special control flow.

Kotlin even tries to tell you about the fact that inline has no useful effect on those functions. The JIT compiler inlines them anyway. The only thing that changes is that it can not collect as much data about the functions which could even decrease performance.